common/decisions/wc_major_zul_decisions.txt

```diff
@@ -0,0 +1,102 @@
+﻿restore_empire_of_zul_decision = {
+	ai_check_interval = 120
+	major = yes
+	# PIC CREDIT https://phillipzhang.artstation.com/
+	picture = "gfx/interface/illustrations/decisions/zulian_restoration.dds"
+	desc = restore_empire_of_zul_decision_desc
+	selection_tooltip = restore_empire_of_zul_decision_tt
+
+	is_shown = {
+		culture = { has_cultural_pillar = heritage_zulite }
+		NOT = {
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:restore_empire_of_zul_decision
+			}
+		}
+		OR = {
+			title:e_amani.holder ?= root
+			title:e_drakkari.holder ?= root
+			title:e_gurubashi.holder ?= root
+			title:e_zandalari.holder ?= root
+			title:e_zulfarrak.holder ?= root
+		}
+	}
+
+	cost = {					
+		gold = 750
+		prestige = 1000
+	}
+	
+	is_valid = {
+		realm_size >= 40
```

Comment:
```suggestion
```
your realm size must be more than 40 anyway, since you have to control all those duchies